src/error.rs

```diff
@@ -1,14 +1,14 @@
 use std::io;
 
 /// Errors that may occur during mocking.
-#[derive(Debug)]
+#[derive(PartialEq, Clone, Debug)]
```

Comment:
```suggestion
#[derive(PartialEq, Clone, Copy, Debug)]
```
Could we make the error types `Copy`? The types are trivial and furthermore [`std::io::ErrorKind`](https://doc.rust-lang.org/std/io/enum.ErrorKind.html#impl-Copy) also is.

Reply:
Right now it is, but if we add more error variants in the future it might become non-copy.

(On the other hand it's just a testing framework, so that might be fine. I'm not sure what would be best.)

Reply:
I see. I can also live with the clone at least for now.